master/master.cfg

```diff
@@ -190,7 +190,8 @@ builders = [
     ("PPC64LE Fedora", "edelsohn-fedora-ppc64le", UnixBuild, STABLE),
     ("s390x SLES", "edelsohn-sles-z", UnixBuild, STABLE),
     ("s390x Debian", "edelsohn-debian-z", UnixBuild, STABLE),
-    ("s390x RHEL", "edelsohn-rhel-z", UnixBuild, STABLE),
+    ("s390x RHEL7", "edelsohn-rhel-z", UnixBuild, STABLE),
```

Comment:
The change here is for the builder name but not the worker. Maybe also change the worker name? Not sure if it's gonna require any changes to the existing config in the worker though.

Reply:
I specifically did not want to change the worker name and lose history.  I simply wanted to change the description to clarify versus the new RHEL8 worker.

Reply:
That is reasonable yeah